prefect_github/credentials.py

```diff
@@ -32,14 +41,18 @@ def get_endpoint(self) -> HTTPEndpoint:
             @flow
             def example_get_endpoint_flow():
                 token = "consumer_key"
-                github_credentials = GitHubCredentials(token)
+                github_credentials = GitHubCredentials(token=token)
```

Comment:
I don't see `token` declared above this line.
```suggestion
                github_credentials = GitHubCredentials(token="ghu_xxxxxxxx")
```

Reply:
token is declared I think, but I updated the generator to replace the "consumer_key" string